workflows/database_params.go

```diff
@@ -0,0 +1,44 @@
+package workflows
+
+import (
+	"fmt"
+	"github.com/stelligent/mu/common"
+)
+
+// DatabaseSetPassword sets a database password for an environment
+func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceNameIn string, newPassword string) Executor {
+	return func() error {
+		workflow := new(databaseWorkflow)
+		workflow.codeRevision = ctx.Config.Repo.Revision
+		workflow.repoName = ctx.Config.Repo.Slug
+		serviceName := serviceNameIn
+		if serviceName == "" {
+			serviceName = workflow.serviceName
```

Comment:
Does this work?  this doesn't appear to be initialized anywhere.

Also, the workflow is being created by not executed...this isn't following the same pattern as other commands.